docs/upgrades.md

```diff
@@ -0,0 +1,318 @@
+   pulpcore's latest version, such as by reading the release announcement email thread for
+   pulpcore's latest version, reading the plugins' README, or reading their `setup.py`.
+1. If they are not all compatible yet, try researching the **previous** minor release (`3.9`) of the installer
+   and pulpcore. Confirm that there exist stable releases of your desired plugins that are compatible
+   with said release. If there are none, try the minor release before that (`3.8`) and repeat.
+1. Once a compatible pulpcore version and plugin versions are found, Set `pulp_install_plugins`
+   with each plugin name listed as a key, and with each plugin having a key under it called `version` set
+   to the minor version (3.y) as the value (with quotes).
+1. Update or downgrade your installer to the latest version (3.y.z) of the pulpcore minor release (3.y) you
+   need. (The installer's minor version will control what minor version of pulpcore gets installed.)
+1. Run `pulp_installer`.
+1. Make sure to save your variables/playbook for later usage.
+
+Commmand to install pulp_installer's latest version:
+```
+ansible-galaxy collection install --force pulp.pulp_installer
+```
+Example commmand to install a specific version of pulp_installer:
+```
+ansible-galaxy collection install --force pulp.pulp_installer:3.9.1
+```
+Example `pulp_install_plugins`:
+```
+  vars:
+    pulp_install_plugins:
+      pulp-container:
+        version: "2.8"
+      pulp-file:
+        version: "1.9"
+      pulp-rpm:
+        version: "3.15"
+```
+
+#### **Updating your installation:**
+
+    Note: These are update instructions, not upgrade instructions. They will keep you on the same pulpcore
+    minor version (3.y) as initially installed.
+
+1. [Update](https://galaxy.ansible.com/pulp/pulp_installer) `pulp_installer` to the latest patch release (3.y.z)
+   within the minor release (3.y) that was used for the initial installation.
+1. Set `pulpcore_update` to `true`.
+1. Set `pulp_install_plugins` with each **currently installed** plugin's name listed as a key.
+   For each each plugin add a key under it
+   called `version` set to the currently installed `x.y` plugin version. Add a key under the plugin name
+   called `upgrade` set to
+   the value `true`. (This combination will will update the plugin to the latest patch release.)
+1. Re-run `pulp_installer`.
+1. Make sure to save your variables/playbook for later usage.
+
+Example commmand to update to a specific version of pulp_installer:
+```
+ansible-galaxy collection install --force pulp.pulp_installer:3.9.2
+```
+
+Example `pulp_install_plugins`:
+```
+  vars:
+    pulpcore_update: true
+    pulp_install_plugins:
+      pulp-container:
+        version: "2.8"
+        upgrade: true
+      pulp-file:
+        version: "1.9"
+        upgrade: true
+      pulp-rpm:
+        version: "3.15"
+        upgrade: true
+```
+
+#### **Upgrading your installation:**
```

Comment:
the title is the same as https://github.com/pulp/pulp_installer/pull/1123/files#diff-2b35cda5da63a0dd7e1834e0063af64826711909facdb8a56fafece345fa24feR70
what about: "_Upgrading your installation to a specific minor release_" (for this line) and
"_Upgrading your installation to the latest release_" (to the other)

Reply:
Done.